ehr_part3.py

```diff
@@ -0,0 +1,63 @@
+import ehr_analysis
+from datetime import datetime
+from typing import List
+
+"""
+At minimum you should end up with:
+
+* a Patient class with:
+  * instance attributes for gender, DOB, race, etc.
+  * a [property](https://docs.python.org/3/library/functions.html#property) called `age`
+* a Lab class with:
+  * instance attributes for value, units, etc.
+
+"""
+
+class Patient:
+    def __init__(self, PatientID: str) -> None:
+        self.patients = ehr_analysis.parse_patient("test_patient.txt")
```

Comment:
The user of the library should be able to specify their own file. We want to modify the `parse_patient` function to produce these Patient objects.

Reply:
This issue is solved.